swatch-metrics/src/main/java/com/redhat/swatch/metrics/service/PrometheusMeteringController.java

```diff
@@ -239,6 +239,35 @@ private void createEventFromDataAndSend(
       role = null;
     }
 
+    boolean is3rdPartyMigrated = Boolean.parseBoolean(labels.get("conversions_success"));
+
+    var matchingTags =
```

Comment:
Are we certain that we will only receive PaygEligible products? I don't see metered flag used here so, nonpayg products not fall within this flow?

Reply:
Yes, we're certain that we only receive PaygEligible products.  Any metrics we grab from prometheus are metered products.

Reply:
Awesome! Thanks for the confirmation